src/interactivity/win32/windowproc.cpp

```diff
@@ -226,18 +226,15 @@ using namespace Microsoft::Console::Types;
         _UpdateSystemMetrics();
         s_ReinitializeFontsForDPIChange();
 
-        if (IsInFullscreen())
-        {
-            // If we're a full screen window, completely ignore what the DPICHANGED says as it will be bigger than the monitor and
-            // instead just ensure that the window is still taking up the full screen.
-            SetIsFullscreen(true);
-        }
-        else

```

Comment:
Will any of these changes regress our windowing downlevel from 21xxx? Or is this "more correct" everywhere?

Reply:
Fair concern. The short answer is 'yes', this change should work properly on all down-level releases.

Re the WM_DPICHANGED message, whenever this message is sent (from any release) anything other than calling SetWindowPos with the RECT from lParam simply isn't supported. If this was ever necessary, it was because of a bug that (I believe) has since been fixed.

I'd be happy to convince you further (either in the open or in a teams chat) :).